core/src/themes/md/default.tokens.ts

```diff
@@ -76,4 +88,119 @@ export const defaultTheme: DefaultTheme = {
     xxxl: 'var(--ion-radii-900)',
     xxxxl: 'var(--ion-radii-full)',
   },
+
+  components: {
+    IonChip: {
+      margin: '4px',
+      paddingVertical: '6px',
+      paddingHorizontal: '12px',
```

Comment:
```suggestion
      padding: {
        vertical: '6px',
        horizontal: '12px',
      }
```

Would it not be cleaner to do it as objects?